src/apns_connection.erl

```diff
@@ -332,10 +339,9 @@ get_device_path(DeviceId) ->
 add_authorization_header(Headers, Token) ->
   Headers#{apns_auth_token => <<"bearer ", Token/binary>>}.
 
--spec push(pid(), apns:device_id(), apns:headers(), notification()) ->
+-spec push(pid(), apns:device_id(), apns:headers(), notification(), integer()) ->
```

Comment:
Line 342 is too long: -spec push(pid(), apns:device_id(), apns:headers(), notification(), integer()) ->.